FilmFestivalLoader/Shared/web_tools.py

```diff
@@ -38,6 +38,12 @@ def get_netloc(url):
     return urlunparse([obj.scheme, obj.netloc, '', '', '', ''])
 
 
+def paths_eq(url, other_url):
+    url_path = urlparse(url)[2]     # <scheme>://<netloc>/<path>;<params>?<query>#<fragment>
+    other_path = urlparse(other_url)[2]
```

Comment:
This doesn't work!
`urlparse()` delivers an object with url parts as attributes, not an iterable.

Please change into `url parse(url).path` en please test.

Reply:
Done.